common/src/main/java/com/revenuecat/purchases/common/BillingAbstract.kt

```diff
@@ -78,7 +78,7 @@ abstract class BillingAbstract {
         activity: Activity,
         appUserID: String,
         storeProduct: StoreProduct,
-        purchaseOption: PurchaseOption,
+        purchaseOption: PurchaseOption?,
```

Comment:
So this is assuming OTP won't have a `PurchaseOption` but this is not completely clear. See https://github.com/RevenueCat/purchases-android/pull/663#discussion_r1015103001. Depending on that conversation, we might have to change this slightly, but it won't be a big change since `PurchaseOption` won't be required for OTP.